facades/PC/src/main/java/org/terasology/engine/Terasology.java

```diff
@@ -77,7 +77,7 @@
  * <tr><td>-headless</td><td>Start headless.</td></tr>
  * <tr><td>-loadlastgame</td><td>Load the latest game on startup.</td></tr>
  * <tr><td>-noSaveGames</td><td>Disable writing of save games.</td></tr>
- * <tr><td>-noCrashReport</td><td>Disable crash reporting.</td></tr>
+ * <tr><td>-noCrashReport</td><td>Disable crash reporting when having a crash.</td></tr>
```

Comment:
My thought was emphasizing the flag doesn't disable the usage of the CR for issue reporting (just on triggering actual crashes) ... but yeah this line as written kinda ends up working anyway, since the new option is indeed more about issue/feedback reporting, not crashes :-)